### ISSUE TITLE
bug: Plaid transactions sync error

### ISSUE THREAD

**Issue by nameduser0** [frappe/erpnext#24675]
## Description of the issue
Plaid transactions sync error
blame #20659 😊

## Context information (for bug reports)

**Output of `bench version`**
```
ERPNext: v12.9.4 (stable-12)
Frappe Framework: v12.6.2 (stable-12)
```

## Steps to reproduce the issue

1. Set up Company
2. Set up Plaid to sync all accounts every hour
3. Set up bank account via Plaid (don't forget to set the integration start date on the bank account)

### Observed result
The below error will be logged in ERPNext `Error Log` doctype

### Expected result
No errors

### Stacktrace / full error message

```
Plaid transactions sync error
Traceback (most recent call last):
  File "/home/erp/production/apps/erpnext/erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py", line 139, in sync_transactions
    for transaction in reversed(transactions):
TypeError: 'NoneType' object is not reversible
```

**Comment by nateriches** [comment]
I'm encountering the same issue with 13.2.1. Initial sync is fine, but afterward this error persists.

**Comment by nameduser0** [comment]
@nateriches Fixed with #25662
